autogluon/task/object_detection/detector.py

```diff
@@ -202,12 +192,11 @@ def load(cls, checkpoint):
         model_params = state_dict['model_params']
         classes = state_dict['classes']
 
-        model = get_network(args.meta_arch, args.net, classes, mx.cpu(0))
+        model = get_network(args.meta_arch, args.net, classes, ctx=mx.cpu(0), syncbn=args.syncbn)
```

Comment:
if cpu is used, syncbn should never become true?

Reply:
i believe syncbn works with cpu as well